### DISCUSSION TITLE
Ability to set MAP api in Settings page inside HASSIO container

### DISCUSSION THREAD

**Discussion by marcodallago** [flobz/psa_car_controller#318] [Ideas]
Since it's quite difficult to access the config inside from Home Assistant installed via Hassio, I'd suggest to add the option to set map API key inside a new tab in the newly settings page! 

Also, setting price for kWh could be useful

Moreover could also be interesting to set ABPR api, instead using terminal. 

Thanks 🙏

**Comment by kurim** [comment]
Yes, your right.
Currently you need to use this was:
Add this repo: https://github.com/MikeJMcGuire/HASSAddons
Install Portainer and open a console to the specific container
